src/Model.test.ts

```diff
@@ -1588,7 +1646,7 @@ describe('Model', () => {
         eventBufferOptions: defaultEventBufferOptions,
       },
     );
-    model.sync();
+    await model.sync();
```

Comment:
@zknill I saw your name on this test. After I made my changes I was getting a random failure from this test I think its cause we call this sync function but don't await. Can you let me know if missing the await was on purpose. Thanks

Reply:
I probably just missed it when I wrote the test 👌 I don't see an obvious reason for it to be missing.